wpilibj/src/main/java/edu/wpi/first/wpilibj/Vector.java

```diff
@@ -0,0 +1,65 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
This feels like it should be in a util or math package.  In particular because "Vector" is an overloaded word, particularly in C++.

Reply:
We could also stick it in the drive package depending what else might use it.